asmcomp/cmm.mli

```diff
@@ -68,6 +68,17 @@ val ge_component
   -> machtype_component
   -> bool
 
+type exttype =
+  | XInt                                (**r OCaml value, word-sized integer *)
+  | XInt32                              (**r 32-bit integer *)
+  | XInt64                              (**r 64-bit integer  *)
+  | XFloat                             (**r double-precision FP number  *)
```

Comment:
The formatting here seems off, with a missing space after `XFloat`, and varying numbers of spaces before the closing comment delimiters...

Reply:
Right.  Fixed.